actionview/lib/action_view/dependency_tracker.rb

```diff
@@ -1,9 +1,12 @@
 require 'thread_safe'
+require 'action_view/path_set'
 
 module ActionView
   class DependencyTracker # :nodoc:
     @trackers = ThreadSafe::Cache.new
 
+    mattr_accessor(:view_paths) { ActionView::PathSet.new }
```

Comment:
Good call here, @rafaelfranca. The digestor gets the `lookup_context` injected through the finder argument, so we can get the `view_paths` from there.

The only problem, is that we have to add a third argument to the tracker's `call` method, which other tracker's definitely don't support yet. Since I don't know of a reliable way to check how many args a method takes, I've added an option trackers should specify when registering.